tests/compiler/closure.ts

```diff
@@ -21,4 +21,51 @@ function testLet(): (value: i32) => i32 {
 }
 testLet();
 
+//function nestedExecutionTest(arg: i32): i32 {
+  //var x = 7;
+  //var f = complexCreateClosure(arg);
+  //var g = (fn: (x3: i32) => i32): i32 => {
+    //var first = fn(arg);
+    //return x;
+  //}
+  //return g(f);
+//}
+//nestedExecutionTest(1);
```

Comment:
Accidentally commited, just wanted to get up what I had while I'm still trying to get tests working